packages/mobile/src/transactions/CeloTransferFeedItem.tsx

```diff
@@ -0,0 +1,96 @@
+// VIEW that's used on the CELO screen activity feed for CELO transfers (in or out).
+
+import Touchable from '@celo/react-components/components/Touchable'
+import colors from '@celo/react-components/styles/colors.v2'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import variables from '@celo/react-components/styles/variables'
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { CeloExchangeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { TransferItemFragment } from 'src/apollo/types'
+import CurrencyDisplay from 'src/components/CurrencyDisplay'
+import { Namespaces } from 'src/i18n'
+import { navigateToPaymentTransferReview } from 'src/transactions/actions'
+import { TransactionStatus } from 'src/transactions/types'
+import { getDatetimeDisplayString } from 'src/utils/time'
+
+type Props = TransferItemFragment & {
+  status?: TransactionStatus
+}
+
+export function CeloTransferFeedItem(props: Props) {
+  const { t, i18n } = useTranslation(Namespaces.walletFlow5)
+  const { address, amount, comment, status, timestamp, type } = props
+  const onPress = () => {
+    ValoraAnalytics.track(CeloExchangeEvents.celo_transaction_select)
+
+    navigateToPaymentTransferReview(type, timestamp, {
+      address,
+      comment,
+      amount,
+      type,
+      // fee TODO: add fee here.
+    })
+  }
+
+  const dateTimeFormatted = getDatetimeDisplayString(timestamp, i18n)
+  const isPending = status === TransactionStatus.Pending
+  const isWithdrawal = new BigNumber(amount.value).isNegative()
+
+  return (
+    <Touchable onPress={onPress}>
+      <View style={styles.container}>
+        <View style={styles.firstRow}>
+          <View style={styles.desc}>
+            <Text style={styles.txMode}>
+              {t(isWithdrawal ? 'feedItemGoldWithdrawal' : 'feedItemGoldReceived', {
+                address: `${address.slice(0, 6)}...${address.slice(-4)}`,
```

Comment:
Do you think we can use https://github.com/celo-org/celo-monorepo/blob/4773390237b68dc71c14f85d60a4bca197b33468/packages/mobile/src/components/ShortenedAddress.tsx#L9-L11 here?

Reply:
Problem is that since this is an object if we pass it to the `t` function is shows as `[Object object]`. I did however extract the formatting function and reused it so that we don't have the formatting code repeated :)